docs/layers/learning_layers.rst

```diff
@@ -0,0 +1,426 @@
+are initialized with He normal initialization and the bias weights are
+initialized to zero.
+
+Arguments:
+
+   :num_dims:
+
+       (``int64``) Number of spatial dimensions
+
+       The first data dimension is treated as the channel dimension, and
+       all others are treated as spatial dimensions (recall that the
+       mini-batch dimension is implicit).
+
+   :num_output_channels:
+
+       (``int64``) Channel dimension of output tensor
+
+   :has_vectors:
+
+       (``bool``) Whether to use vector-valued options
+
+       If true, then the pooling is configured with ``conv_dims``,
+       ``conv_pads``, ``conv_strides``, ``conv_dilations``. Otherwise,
+       ``conv_dims_i``, ``conv_pads_i``, ``conv_strides_i``,
+       ``conv_dilations_i``.
+
+   :conv_dims:
+
+       (``string``) Convolution kernel dimensions (vector-valued)
+
+       Space-separated list of integers, one for each spatial
+       dimension. Used when ``has_vectors`` is enabled.
+
+   :conv_pads:
+
+       (``string``) Convolution padding (vector-valued)
+
+       Space-separated list of integers, one for each spatial
+       dimension. Used when ``has_vectors`` is enabled.
+
+   :conv_strides:
+
+       (``string``) Convolution strides (vector-valued)
+
+       Space-separated list of integers, one for each spatial
+       dimension. Used when ``has_vectors`` is enabled.
+
+   :conv_dilations:
+
+       (``string``) Convolution dilations (vector-valued)
+
+       Space-separated list of integers, one for each spatial
+       dimension. Used when ``has_vectors`` is enabled. Defaults to
+       dilations of 1, i.e. undilated convolution.
+
+   :conv_dims:
+
+       (``int64``) Convolution kernel size (integer-valued)
+
+       Used when ``has_vectors`` is disabled.
+
+   :conv_pads_i:
+
+       (``int64``) Convolution padding (integer-valued)
+
+       Used when ``has_vectors`` is disabled.
+
+   :conv_strides_i:
+
+      (``int64``) Convolution stride (integer-valued)
+
+      Used when ``has_vectors`` is disabled.
+
+   :conv_dilations_i:
+
+      (``int64``, optional) Convolution dilation (integer-valued)
+
+      Default: 1
+
+      Used when ``has_vectors`` is disabled.
+
+   :has_bias: (``bool``) Whether to apply per-channel bias
+
+   :num_groups:
+
+      (``int64``, optional) Number of channel groups for grouped
+      convolution
+
+      Default: 1
+
+   :conv_tensor_op_mode:
+
+      (``ConvTensorOpsMode``) Special behavior with FP16 tensor cores
+
+      Ignored for non-GPU layers.
+
+Deprecated and unused arguments:
+
+   :weight_initialization: (``string``)
+
+   :bias_initial_value: (``double``)
+
+   :l2_regularization_factor: (``double``)
+
+:ref:`Back to Top<learning-layers>`
+
+________________________________________
+
+
+.. _Deconvolution:
+
+----------------------------------------
+Deconvolution
+----------------------------------------
+
+Deconvolution Layer. Reverses the effects of convolution.
```

Comment:
```suggestion
Deconvolution Layer. Transpose of convolution.
```
Once https://github.com/LLNL/lbann/pull/1997 is merged, the text should be ported here.